apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -141,17 +153,21 @@ export default function BaseMegaMenu() {
         className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10"
       >
         <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
-          <a href="/" aria-label="SF Homepage" className="inline-block text-white mr-2 lg:mr-10">
+          <a
+            href="/"
+            aria-label="SF Homepage"
+            className="inline-block mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+          >
             <picture>
-              <source srcSet={brandLogo.src} media="(min-width: 767px)" />
-              <img src={brandLogoSign.src} alt="Sf Logo" className="w-8 h-8 mr-4 md:w-[12.5rem] md:h-[1.75rem]" />
+              <source srcSet={brandLogo.src} media="(min-width: 1025px)" />
+              <img src={brandLogoSign.src} alt="Sf Logo" className="w-8 h-8 lg:w-[12.5rem] lg:h-[1.75rem]" />
             </picture>
           </a>
-          <nav className="flex w-full justify-between flex-nowrap" aria-label="SF Navigation">
+          <nav className="flex justify-between w-full flex-nowrap" aria-label="SF Navigation">
             <ul>
               <li role="none">
                 <SfButton
```

Comment:
I think this button should get `square` prop on mobile. Right now its padding is a bit too big for an icon.
<img width="308" alt="Screenshot 2023-04-21 at 12 10 59" src="https://user-images.githubusercontent.com/34419118/233611618-84dfe9f8-9140-40d0-bf81-c786be6a71f6.png">

Reply:
It's exactly like on figma:  
<img width="176" alt="Screenshot 2023-04-24 at 09 30 26" src="https://user-images.githubusercontent.com/46591755/233928592-cebff684-c495-40e7-9bf9-6b8eb7c892eb.png">

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2622/commits/2a200c8395073393be3f5853f9162e9c2d8ea3d3